### ISSUE TITLE
Revising of "Pajaro" tema in mark-up

### ISSUE THREAD

**Issue by aldobarriente** [Multepal/popolwuj#40]
The _pajaro_ tema is appears inconsistently throughout the mark-up of the text. It would be good maintenance to define and fix its usage.

**Comment by allisonmargaret** [comment]
Yes, that was made by a student who unfortunately did not follow the course encoding guidelines.

Allison Margaret Bigelow
433 New Cabell Hall / PO Box 400777
Department of Spanish, Italian, and Portuguese
University of Virginia
Charlottesville, VA 22904
amb8fk@virginia.edu * +1.434.243.3371 (p)
http://spanitalport.as.virginia.edu/people/profile/amb8fk
________________________________
